scripts/fuzz_opt.py

```diff
@@ -1855,22 +1849,50 @@ def handle(self, wasm):
             wasms[wasm_index] = new_name
 
         # Run again, and compare the output
-        optimized_output = run_d8_wasm(wasms[0], args=[wasms[1]])
+        optimized_output = run_bynterp(wasms[0], args=['--fuzz-exec-before', f'--fuzz-exec-second={wasms[1]}'])
         optimized_output = fix_output(optimized_output)
 
         compare(output, optimized_output, 'Two')
 
+        # If we can, also test in V8. We also cannot compare if there are NaNs
+        # (as optimizations can lead to different outputs), and we must
+        # disallow some features.
+        # TODO: relax some of these
+        if NANS or not all_disallowed(['shared-everything', 'strings', 'stack-switching']):
+            return
+
+        output = run_d8_wasm(wasm, args=[second_wasm])
+
+        if output == IGNORE:
+            return
+
+        if output.startswith(INSTANTIATE_ERROR):
```

Comment:
We can probably skip the check below and immediately report a problem, since our interpreter did _not_ trap in instantiation. It also might be worth checking the other direction by running V8 when the interpreter traps in instantiation to make sure V8 also traps in instantiation.

Reply:
Oh, but we do handle IGNORE earlier, so you are right: only deterministic things should remain at this point. I simplified it now.